src/shared_modules/content_manager/src/contentModuleFacade.hpp

```diff
@@ -18,18 +18,58 @@
 
 constexpr auto CONTENT_MODULE_ENDPOINT_NAME {"content"};
 
+/**
+ * @brief ContentModuleFacade class.
+ *
+ */
 class ContentModuleFacade final : public Singleton<ContentModuleFacade>
 {
 private:
     std::unordered_map<std::string, std::unique_ptr<ContentProvider>> m_providers;
     std::shared_mutex m_mutex;
 
 public:
+    /**
+     * @brief Register endpoint to receive initialization of modules from other processes.
+     *
+     */
     void start();
+
+    /**
+     * @brief Clear providers.
+     *
+     */
     void stop();
```

Comment:
Question: are these descriptions correct? They don't seem to match the name of the functions.

Reply:
Yes. I tried to describe what the implementation does:
```C++
void ContentModuleFacade::start()
{
    // Register endpoint to receive initialization of modules from other processes.
    /*    RouterProvider::instance().initLocal(CONTENT_MODULE_ENDPOINT_NAME);
        RouterSubscriber::instance().subscriberLocal(CONTENT_MODULE_ENDPOINT_NAME,
                                                     [&](std::shared_ptr<std::vector<char>> data)
                                                     {
                                                         try
                                                         {
                                                             FactoryDecoder::create(data)->decode();
                                                         }
                                                         catch (const std::exception& e)
                                                         {
                                                             std::cerr << "start: " << e.what() << std::endl;
                                                         }
                                                     });*/
}

void ContentModuleFacade::stop()
{
    std::lock_guard<std::shared_mutex> lock {m_mutex};
    m_providers.clear();
}
```